1-exercises/C-object-methods/exercise-3.js

```diff
@@ -12,11 +12,12 @@ let highScores = {
   100: ["David"],
 };
 
-let highScoreKeys = /* todo */
-let highScoreValues = /* todo */
+let highScoreKeys = Object.keys(highScores); /* todo */
+let highScoreValues = Object.values(highScores);
```

Comment:
To list all names in different arrays  as string, I used chained method for this object values:  .flat().join(' ,  ')

Reply:
@howard-ss Great Job